### DISCUSSION TITLE
Will there be a .NET Core 3.1.426 release in source-build?

### DISCUSSION THREAD

**Discussion by omajid** [dotnet/source-build#3182] [General]
I know that .NET Core 3.1 has reached its end of life. The last release that [Microsoft published was 3.1.426/3.1.32](https://dotnet.microsoft.com/en-us/download/dotnet/3.1). Will there be a source-build tag/release for that here? I don't see it in [source-build the tag list](https://github.com/dotnet/source-build/tags) or in the list of open PRs.

**Comment by crummel** [comment]
Sorry, I'll push this in a bit.  Thanks for the reminder!

**Comment by crummel** [comment]
Sorry for the delay, the PR is https://github.com/dotnet/source-build/pull/3183.

**Reply by omajid** [reply]
Thanks!
